tests/test_toolbars.py

```diff
@@ -488,26 +460,30 @@ def test_view_published_in_toolbar_in_preview_mode_button_url(self):
 
 
 class VersioningPageToolbarTestCase(CMSTestCase):
-
     def _get_toolbar_item_by_name(self, menu, name):
         for item in menu.items:
             if hasattr(item, "name") and item.name == name:
                 return item
         return None
 
-    @override_settings(CMS_LANGUAGES = {1: [{"code": "en", "name": "English"}]})
+    @override_settings(CMS_LANGUAGES={1: [{"code": "en", "name": "English"}]})
     def test_change_language_menu_page_toolbar_one_languages(self):
-        page_content = PageContentWithVersionFactory()
+        page_content = PageContentWithVersionFactory(language="en")
+        page = page_content.page
+
+        page.update_languages(["en"])
+
         request = self.get_page_request(
-            page=page_content.page,
+            page=page,
             path=get_object_edit_url(page_content),
             user=self.get_superuser(),
         )
         request.toolbar.set_object(page_content)
         request.toolbar.populate()
         request.toolbar.post_template_populate()
+
         language_menu = request.toolbar.get_menu(LANGUAGE_MENU_IDENTIFIER)
-        self.assertIsNone(language_menu)
+        self.assertIsNone(language_menu, "Language menu should not be created if the page has only one language.")
```

Comment:
Heads up: we forgot to remove the modifications done to this test case. Just copypaste the code from the master branch to make sure it stays the same: https://github.com/django-cms/djangocms-versioning/blob/master/tests/test_toolbars.py#L499